.azurepipelines/Ubuntu-GCC5.yml

```diff
@@ -17,24 +28,23 @@ schedules:
     - main
   always: true          # Always build, even if no changes
 
-pr:
-- main
-# MU_CHANGE END
-
 jobs:
-- template: templates/pr-gate-build-job.yml
+- template: Jobs/PrGate.yml@mu_devops
   parameters:
-    tool_chain_tag: 'GCC5'
-    vm_image: 'ubuntu-latest'
-    arch_list: "IA32,X64,ARM,AARCH64"
-    # MU_CHANGE
+    arch_list: IA32,X64,ARM,AARCH64
+    do_ci_build: true
+    do_ci_setup: false
     extra_steps:
     - script: sudo apt-get install mingw-w64

```

Comment:
apt vs apt-get?

Based on some quick searches, would it be better to switch over to using apt since ubuntu has that listed as their package manager?  asking because we are linked to ubuntu-latest.

https://ubuntu.com/server/docs/package-management

Reply:
I'm in favor of moving to apt. I think that's something I'd like to do as a follow up change though.

Reply:
This simple installation will have no problem with apt. I'll apply that across all the repos.